Public/Sdk/Public/Transformers/Transformer.ReadPipGraphFragment.dsc

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Transformer {
+
+    /**
+     * Read in a fragment of the graph as a binary file and all all pip from it to the pip graph.
+     * name: Name of this pip graph fragment, used to return a handle to it.
+     * file: Binary file to read.
+     * dependencyNames: All other pip graph fragments this one depends on.
+     * Returns a handle to the fragment just created, which can be used to specify dependencies on other graphs.

```

Comment:
Is it possible to define a pip (by calling the standard `Transformer.execute()`) that depends on a pip from one of these graph fragments?

Reply:
Not currently.  I was anticipating that being a future extension if someone needs it (I don't think anyone will currently).

---
In reply to: [289545145](https://github.com/microsoft/BuildXL/pull/394#discussion_r289545145) [](ancestors = 289545145)

Reply:
What about in the opposite direction: can pips from these graph fragments depend on explicitly defined pips?

Reply:
nope, same reason.  MVP doesn't need it, and we can add it later when needed.

---
In reply to: [289557244](https://github.com/microsoft/BuildXL/pull/394#discussion_r289557244) [](ancestors = 289557244)